test/load.test.ts

```diff
@@ -55,6 +89,39 @@ const EXT_PKG_RESPONSE = {
       canonical: 'http://hl7.org/fhir/extensions',
       description: 'None',
       url: 'https://packages2.fhir.org/packages/hl7.fhir.uv.extensions/4.5.6-test'
+    },
+    '1.1.0': {
+      name: 'hl7.fhir.uv.extensions',
+      date: '2023-03-26T08:46:31-00:00',
+      version: '1.1.0',
+      fhirVersion: '??',
+      kind: '??',
+      count: '18',
+      canonical: 'http://hl7.org/fhir/extensions',
+      description: 'None',
+      url: 'https://packages2.fhir.org/packages/hl7.fhir.uv.extensions/1.1.0'
+    },
+    '1.1.2': {
```

Comment:
Is there a reason you are keeping the `hl7.fhir.uv.extensions` and `hl7.terminology.r4` versions in sync like this? I feel like the unit tests would be more robust if we expect different latest versions from each package instead of the same latest versions from each package. Just in case there was every a bug that mixed packages up.

Reply:
Oh, good point! No real reason, I just thought it simplified things for my mind reading the tests, but I think that's a good reason to not over simplify it. I'll revert these ones back to a v2 or v3 since that makes sense for the latest version, which is a v4.

Reply:
Updated in [b53ef0a](https://github.com/standardhealth/fhir-package-loader/pull/15/commits/b53ef0aa0ad1e6b1b6377ce9ce33d6eda38f582d)